tests/test_expected.py

```diff
@@ -303,3 +303,36 @@ def test_trans_expected_regions_cli(request, tmpdir):
             # atol=0,
             equal_nan=True,
         )
+
+
+def test_logbin_expected_cli(request, tmpdir):
```

Comment:
added a stupid logbin expected test - just to make sure the new CLI runs, and that it generates output - output contents are not checked